integration-tests/rpc_integration_test.go

```diff
@@ -0,0 +1,465 @@
+	wg := &sync.WaitGroup{}
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		startStandaloneNode(t, ctx, count, logMessages)
+	}()
+
+	_, err := waitForLogSubstring(ctx, logMessages, "started RPC server")
+	require.NoError(t, err, "RPC server didn't start")
+
+	ethClient := ethclient.NewClient(ethRPCClient)
+	signedTestOrder := scenario.CreateZRXForWETHSignedTestOrder(t, ethClient, makerAddress, takerAddress, wethAmount, zrxAmount)
+	expectedOrderHash, err := signedTestOrder.ComputeOrderHash()
+	require.NoError(t, err, "could not compute order hash for standalone order")
+
+	expectedFillableTakerAssetAmount := signedTestOrder.TakerAssetAmount
+
+	client, err := rpc.NewClient(standaloneRPCEndpoint + strconv.Itoa(rpcPort+count))
+	require.NoError(t, err)
+
+	signedTestOrders := []*zeroex.SignedOrder{signedTestOrder}
+	validationResponse, err := client.AddOrders(signedTestOrders)
+	require.NoError(t, err)
+	assert.Len(t, validationResponse.Accepted, 1)
+	assert.Len(t, validationResponse.Rejected, 0)
+
+	acceptedOrderInfo := validationResponse.Accepted[0]
+
+	// Reset the hash so that the orders can be compared
+	signedTestOrder.ResetHash()
+
+	assert.Equal(t, expectedOrderHash, acceptedOrderInfo.OrderHash, "orderHashes did not match")
+	assert.Equal(t, signedTestOrder, acceptedOrderInfo.SignedOrder, "signedOrder did not match")
+	assert.Equal(t, expectedFillableTakerAssetAmount, acceptedOrderInfo.FillableTakerAssetAmount, "fillableTakerAssetAmount did not match")
+
+	// Cancel the context and wait for all outstanding goroutines to finish.
+	cancel()
+	wg.Wait()
+}
+
+// TODO(jalextowle): Since the uuid creation process is inherently random, we
+//                   can't meaningfully sanity check the returnedSnapshotID in
+//                   this test. Unit testing should be implemented to verify that
+//                   this logic is correct, if necessary.
+func TestGetOrdersSuccess(t *testing.T) {
+	teardownSubTest := setupSubTest(t)
+	defer teardownSubTest(t)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	// NOTE(jalextowle): Cancel in case execution stops before other cancel is called below.
+	defer cancel()
+
+	// Remove the old database and p2p files.
+	removeOldFiles(t, ctx)
+
+	buildStandaloneForTests(t, ctx)
+
+	// logMessages is a channel through which log messages from the
+	// node will be sent. We use a large buffer so it doesn't cause
+	// goroutines to block.
+	logMessages := make(chan string, 1024)
+
+	ethClient := ethclient.NewClient(ethRPCClient)
+	signedTestOrder := scenario.CreateZRXForWETHSignedTestOrder(t, ethClient, makerAddress, takerAddress, wethAmount, zrxAmount)
+	expectedOrderHash, err := signedTestOrder.ComputeOrderHash()
+	require.NoError(t, err, "could not compute order hash for standalone order")
+
+	expectedFillableTakerAssetAmount := signedTestOrder.TakerAssetAmount
+
+	expectedPage := 0
+	expectedPerPage := 5
+	expectedSnapshotID := ""
+
+	// Set up the dummy handler with an addOrdersHandler
```

Comment:
The code doesn't match the comment. "dummy handler" was only used in the old RPC tests in __rpc/client_server_test.go__, which was removed in this PR. This also applies to a few other places.